src/SprykerEco/Zed/ArvatoRss/Business/Api/Mapper/RiskCheckRequestMapper.php

```diff
@@ -29,23 +31,31 @@ class RiskCheckRequestMapper implements RiskCheckRequestMapperInterface
      */
     protected $billingCustomerMapper;
 
+    /**
+     * @var \SprykerEco\Zed\ArvatoRss\Business\Api\Mapper\Aspect\DeliveryCustomerMapperInterface
+     */
+    protected $deliveryCustomerMapper;
+
     /**
      * @var \SprykerEco\Zed\ArvatoRss\Business\Api\Mapper\Aspect\OrderMapperInterface
      */
     protected $orderMapper;
 
     /**
-     * @param \SprykerEco\Zed\ArvatoRss\Business\Api\Mapper\Aspect\IdentificationMapperInterface $identificationMapper
-     * @param \SprykerEco\Zed\ArvatoRss\Business\Api\Mapper\Aspect\BillingCustomerMapperInterface $billingCustomerMapper
-     * @param \SprykerEco\Zed\ArvatoRss\Business\Api\Mapper\Aspect\OrderMapperInterface $orderMapper
+     * @param IdentificationMapperInterface $identificationMapper
```

Comment:
Please run sniffer in forced mode to fix class names - should be fully qualified

Reply:
It's "fix" mode :)